pkg/apis/events/v1/implements_test.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
this copyright seems with a weird formatting. Can you use the one in the previous file for example